SETUP/db_schema.sql

```diff
@@ -701,6 +701,7 @@ CREATE TABLE `user_project_info` (
   `iste_sr_available` tinyint(1) NOT NULL default '0',
   `iste_ppv_enter` tinyint(1) NOT NULL default '0',
   `iste_posted` tinyint(1) NOT NULL default '0',
+  `iste_sr_posted` tinyint(1) NOT NULL default '0',
```

Comment:
Echoing @srjfoo's comment in slack `sr_reported` or even `sr_uploaded` would probably be clearer.